CHANGELOG.adoc

```diff
@@ -39,6 +39,10 @@ Improvements::
   * use shorthands %F and %T instead of %Y-%m-%d and %H:%M:%S to format time
   * read file in binary mode whenever contents are being normalized
 
+Build / Infrastructure::
+
+  * replace thread_safe with concurrent-ruby (PR #2822) (*@junaruga*)
+
```

Comment:
This part should be changed? This PR is not only for Build / Infrastructure?

Reply:
Since it changes the APIs used by this project, you're right. It's not just a dependency change. Therefore, it should now be under Improvements.

Reply:
Okay, I moved it to Improvement.

Reply:
Thanks!